packages/slider/test/slider.test.ts

```diff
@@ -1604,4 +1603,75 @@ describe('Slider', () => {
         await elementUpdated(el);
         expect(el.values).to.deep.equal({ a: 10, b: 20, c: 29 });
     });
+    it('resets to default value on double click after moving pointer', async () => {
+        let pointerId = -1;
+        const el = await fixture<Slider>(
+            html`
+                <sp-slider
+                    style="width: 100px"
+                    value="50"
+                    default-value="50"
+                ></sp-slider>
+            `
+        );
+        await elementUpdated(el);
+        expect(el.value, 'initial').to.equal(50);
+        expect(pointerId).to.equal(-1);
+        const handle = el.shadowRoot.querySelector('.handle') as HTMLDivElement;
+        el.track.setPointerCapture = (id: number) => (pointerId = id);
+        el.track.releasePointerCapture = (id: number) => (pointerId = id);
```

Comment:
If we use `sendMouse()` all the way through the test here, instead of synthetically dispatching events. Then we should be able to exclude this part of the test all together. Here we are mimicking the default pointerCapture functionality, which the actual pointer interactions in `sendMouse()` do not need.